src/System.Diagnostics.StackTrace/src/System.Diagnostics.StackTrace.sln

```diff
@@ -0,0 +1,34 @@
+﻿
```

Comment:
Yes. I didn’t really mean to add it.

From: Matt Ellis [mailto:notifications@github.com]
Sent: Friday, May 20, 2016 11:50 AM
To: dotnet/corefx <corefx@noreply.github.com>
Cc: Mike McLaughlin <mikem@microsoft.com>; Assign <assign@noreply.github.com>
Subject: Re: [dotnet/corefx] Add symbol support helper class for mscorlib's StackTrace. (#8670)


In src/System.Diagnostics.StackTrace/src/System.Diagnostics.StackTrace.sln<https://github.com/dotnet/corefx/pull/8670#discussion_r64089807>:

> @@ -0,0 +1,34 @@

> +﻿

Can you just delete this SLN? We usually put slns up a level and they include everything.

—
You are receiving this because you were assigned.
Reply to this email directly or view it on GitHub<https://github.com/dotnet/corefx/pull/8670/files/c97f9137ba65fbbaff6850d558df7778d201df70#r64089807>